documentacion/posintegrado/README.md

```diff
@@ -0,0 +1,275 @@
+# POS Integrado
+
+## Cómo empezar
+
+El SDK para POS Integrado, cuenta de 2 partes, la primera pieza es una librería/SDK escrito en C,
+el cual es multi plataforma, y debe ser instalado en tu maquina para poder ser utilizado por la
+segunda pieza, la cual mediante un Wrapper, accede a las funciones disponibles en la librería C.
+
+Adicionalmente, necesitaras tener instalados los drivers correspondientes a tu tarjeta de
+puerto serial o adaptador USB Serial.
+
+<aside class="notice">
+La comunicación con el POS Integrado se realiza mediante puerto serial RS232.
+</aside>
+
+<aside class="notice">
+Tu eres el responsable de instalar el driver correcto para tu tarjeta o adaptador serial.
+</aside>
+
+### LibSerialPort
+
+El SDK depende de [libSerialPort](https://sigrok.org/wiki/Libserialport) para la comunicación serial.
+
+Puedes obtener el código desde el repositorio usando git:
```

Comment:
Do people need the source? Or is it just for advanced usage? (the content under `documentacion` should be kept as straightforward as possible so that people get to use our SDK in the common way as fast as possible; the `reference` section is where we put all the details and extra info)

Reply:
Related note: I see there are binary distributions of libserialport: https://sigrok.org/wiki/Downloads#Binaries_and_distribution_packages

Reply:
The binaries for libSerialport are only available for Linux. I already tried the windows packages and they are only for the other apps from Sigrock (pulseview and sigrok-cli) which doesn't include libSerialport . 

Maybe we could compile and upload libSerialport dll alongside the Wrapper.

Reply:
Fair enough. We can leave that later for the support/evolution of the product and go live with the "use the source, luke" method :)